dist/quotation.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+class Quotation {
+    constructor() {
+        //dummy info
+        this.id = "ID";
+        this.scheduleAt = new Date();
+        this.serviceType = "Service Type";
+        this.specialRequests = ["sp1", "sp2"];
+        this.expiresAt = new Date();
+        this.priceBreakdown = {
+            base: "15",
```

Comment:
I saw the PriceBreakdown model only have these fields as well.. Aren't we handling the other pricebreakdown fields like vat, insurance, etc?

Reply:
Damn. Good catch! I implemented things in the UML, will add the missing ones